src/frontend/app/features/applications/ssh-application/ssh-application.component.ts

```diff
@@ -35,8 +38,23 @@ export class SshApplicationComponent implements OnInit {
 
   public instanceId: string;
 
+  public breadcrumbs$: Observable<IHeaderBreadcrumb[]>;
+
   @ViewChild('sshViewer') sshViewer: SshViewerComponent;
 
+  private getBreadcrumbs(
+    application: IApp,
+  ) {
+    return [
+      {
+        breadcrumbs: [
+          { value: 'Applications', routerLink: '/applications' },
+          { value: application.name, routerLink: `/applications/${application.cfGuid}/${application.cfGuid}/instances` }
```

Comment:
Wrong link - cfGuid twice - second should be app guid

Reply:
This strangely almost worked. Somehow navigated to the app summary page. Have now fixed